app/models/spotlight/custom_field.rb

```diff
@@ -78,7 +78,15 @@ def field_name
     end
 
     def field_slug
-      configuration['label'].parameterize
+      "#{readonly_marker}#{configuration['label'].parameterize}"
+    end
+
+    def readonly_marker
```

Comment:
`field_prefix`, maybe? I can imagine a similar pattern for e.g. multilingual values.